testing/databaseTest.py

```diff
@@ -135,17 +135,16 @@ def test_kinetics(self):
 
             # these families have some sort of difficulty which prevents us from testing accessibility right now
             difficult_families = ['Diels_alder_addition', 'Intra_R_Add_Exocyclic', 'Intra_R_Add_Endocyclic', 'Retroene']
-            generated_trees = ["R_Recombination"]
 
-            if len(family.forward_template.reactants) < len(family.groups.top) and family_name not in difficult_families:
+            if len(family.forward_template.reactants) < len(family.groups.top) and family_name not family.auto_generated:
```

Comment:
```suggestion
            if len(family.forward_template.reactants) < len(family.groups.top) and family_name not in difficult_families:
```
This line wasn't previously checking for auto-generated trees, but "dificult" families. If you intended to change that behaviour, just explain why in your commit message, otherwise I assume it's an error and this line should be reverted.

Reply:
Sorry that wasn't intended! Thanks!